services/users/users.go

```diff
@@ -32,33 +32,34 @@ func New(s store.UserStore, cfgFn func() *model.Config) *UserService {
 
 // CreateUser creates a user
 func (us *UserService) CreateUser(user *model.User, opts UserCreateOptions) (*model.User, error) {
-	user.Roles = model.SYSTEM_USER_ROLE_ID
-	if opts.Guest {
-		user.Roles = model.SYSTEM_GUEST_ROLE_ID
-	}
+	if !opts.FromImport {
```

Comment:
Wondering if we can just return early if it's coming from the import and directly return `us.createUser(user)`. Mainly to avoid the nesting for the less common case.